auth/src/main/java/com/firebase/ui/auth/ui/provider/ProviderBase.java

```diff
@@ -0,0 +1,30 @@
+package com.firebase.ui.auth.ui.provider;
+
+import android.arch.lifecycle.LiveData;
+import android.content.Intent;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.RestrictTo;
+import android.support.annotation.StringRes;
+
+import com.firebase.ui.auth.IdpResponse;
+import com.firebase.ui.auth.ui.HelperActivityBase;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public abstract class ProviderBase {
```

Comment:
Since this class is only abstract methods ... could it be an `interface` instead?

Reply:
Good catch, I didn't notice!